app/renderer/components/SoundBar/SoundBar.scss

```diff
@@ -1,6 +1,6 @@
 .soundBarContainer {
   margin: 20px 0;
-  .bar {
+  .soundBarContainer__bar {
```

Comment:
You should write `&__bar`, it is same as `.soundBarContainer__bar`, but you haven't to repeat class name again